submissions/zadorogniyandrey/scss/style.scss

```diff
@@ -0,0 +1,182 @@
+@import "nullstyle.scss";
+
+
+.wrapper{
+    max-width: 700px;
+    margin: 0 auto;
+    transition: 0.3s;
+}
+.header{
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    padding: 50px 10px;
+}
+.header__link{
+    color: #000;
+    display: block;
+    border: 2px solid transparent;
+    padding: 5px;
+    border-radius: 5px;
+}
+.header__link:hover{
+    border: 2px solid rgb(94, 170, 233);
+}
+.header__link:focus{
+    border: 2px solid rgb(94, 170, 233);
+}
+.header__info{
+    position: relative;
+    display: flex;
+    align-items: center;
+}
+.header__info-item{
+    margin-right: 20px;
+    cursor: pointer;
+}
+.header__info-item:last-child{
+    margin-right: 0;
+}
+.header__bell{
+    width: 35px;
+    height: 40px;
+}
+.header__bell img{
+    width: 100%;
+    height: 100%;
+}
+.menu{
+    position: relative;
+    display: block;
+    width: 35px;
+    height: 35px;
+    padding: 5px;
+    background-color: transparent;
+    border-radius: 5px;
+    transition: 0.3s;
+    border: 2px solid transparent;
+}
+.menu:hover{
+    border: 2px solid rgb(94, 170, 233);
+}
+.menu__input{
+    appearance: none;
+    position: absolute;
+}
+.menu__input:focus ~ .menu{
+    border: 2px solid rgb(94, 170, 233);
+}
+.menu-img{
+    height: 100%;
+    width: 100%;
+    cursor: pointer;
+}
+.header__admin{
+    width: 60px;
+    height: 60px;
+    border-radius: 50%;
+    transition: 0.3s
+}
+.header__admin img{
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+}
+.menu__input:checked ~ .popup{
+    display: block;
+}
+
+//==================POPUP========================//
+
+.popup{
+    position: absolute;
+    top: 65px;
+    right: 0px;
+    width: 297px;
+    height: 450px;
+    border: 1px solid rgb(116, 116, 116);
+    border-radius: 5px;
+    padding: 10px;
+    box-shadow: 0px 0px 9px rgb(146, 146, 146);
+    display: none;
+    overflow-y: auto;
+}
+.popup__items{
+    display: grid;
+    width: 100%;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 10px;
+    align-items: center;
+    margin-bottom: 10px;
+}
+.popup__item{
+    
+    height: 115px;
+}
+.popup__link{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    border: 2px solid transparent;
+    width: 100%;
+    height: 100%;
+    padding: 5px;
+    border-radius: 5px;
+}
+.popup__link:hover{
+    border: 2px solid rgb(94, 170, 233);
+}
+.popup__link:focus{
+    border: 2px solid rgb(94, 170, 233);
+}
+.popup__img-container{
+    display: flex;
+    align-items: center;
+    flex-grow: 1;
+}
+.popup__img{
+    width: 100%;
+    top: -50%;
```

Comment:
please explain why do you need this

Reply:
> please explain why do you need this

Reply:
> please explain why do you need this

initially the picture was not in the container and I tried to make it vertically in the center